### PR TITLE
Managing Default Payment Options

### PR DESCRIPTION
**Pull request by BNMoyers**
# Description
   This repo deals with setting and updating the adult user's default payment method

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] updated relevant unit tests
- [x] manually tested

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/app/components/dashboard/stripe/card/card.component.tsx [hunk 1]
```diff
@@ -18,27 +28,70 @@ const useStyles = makeStyles((theme) => ({
...
     },
+    chip: {
+        marginLeft: 12,
+    },
 }));
 
-const StripeCard: React.FC<StripeCardProps> = ({ card, onDelete }) => {
-    const { request, response } = useAPI(`/payment/cards/${card.id}`, 'DELETE');
+const StripeCard: React.FC<StripeCardProps> = ({ card, onDelete, onUpdate, defaultCard }) => {
+    console.log('default', defaultCard);
```
Comment (EricBanker12) [inline src/app/components/dashboard/stripe/card/card.component.tsx]: front end console.logs should be removed

File: src/app/components/dashboard/stripe/card/card.component.tsx [hunk 2]
```diff
@@ -18,27 +28,70 @@ const useStyles = makeStyles((theme) => ({
...
+    const { request: updateRequest, response: updateResponse } = useAPI(
+        `/payment/default/${card.id}`,
+        'PUT'
+    );
+    const { request: deleteRequest, response: deleteResponse } = useAPI(
+        `/payment/cards/${card.id}`,
+        'DELETE'
+    );
+    console.log('response', updateResponse);
```
Comment (EricBanker12) [inline src/app/components/dashboard/stripe/card/card.component.tsx]: here too, front end console.logs should be removed

**Review by EricBanker12 — COMMENTED** [review]
The payment-picker.component test file is still blank, so it fails testing. Also a couple console.logs should be removed.

Edit: In manual testing, I was not able to change the default card.

File: src/app/components/dashboard/stripe/card/card.component.tsx (hunk 2 above)
Comment (BNMoyers) [reply src/app/components/dashboard/stripe/card/card.component.tsx]: removed

File: src/app/components/dashboard/stripe/card/card.component.tsx (hunk 1 above)
Comment (BNMoyers) [reply src/app/components/dashboard/stripe/card/card.component.tsx]: removed the unused files and combed through for stray logs (there were none remaining).

**Review by EricBanker12 — COMMENTED** [review]
Crashes on build. You need to also delete the payment-picker reference on line 5 in `src/app/components/dashboard/stripe/index.ts`

**Review by wSedlacek — APPROVED** [review]
Working on Staging. Looks good.

**Pull request merged**